pkg/server/server_operation_test.go

```diff
@@ -0,0 +1,83 @@
+package server
+
+import (
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/goccy/go-json"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/stretchr/testify/assert"
+	"github.com/tbd54566975/ssi-service/pkg/server/router"
+	"github.com/tbd54566975/ssi-service/pkg/service/operation"
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestOperationsAPI(t *testing.T) {
+	t.Run("GetOperations", func(t *testing.T) {
+		t.Run("Returns empty when no operations stored", func(t *testing.T) {
+			s, err := storage.NewBoltDB()
+			assert.NoError(t, err)
+			opRouter := setupOperationsRouter(t, s)
+
+			request := router.GetOperationsRequest{}
+			value := newRequestValue(t, request)
+			req := httptest.NewRequest(http.MethodGet, "https://ssi-service.com/v1/operations", value)
+			w := httptest.NewRecorder()
+
+			assert.NoError(t, opRouter.GetOperations(newRequestContext(), w, req))
+
+			var resp router.GetOperationsResponse
+			assert.NoError(t, json.NewDecoder(w.Body).Decode(&resp))
+			assert.Empty(t, resp.Operations)
+		})
```

Comment:
newline for readability between these plz

Reply:
Done